lib/fetch/index.js

```diff
@@ -0,0 +1,88 @@
+import Prismic from 'prismic.io';
+import { PrismicConfig } from '../config/prismic.js';
+import {
+  sanitizeEvent,
+  sanitizeNews,
+  sanitizeCommunity,
+  sanitizeSpeaker,
+  sanitizeTalk,
+} from './sanitize';
+
+async function fetchOne(docType, sanitizeFunction, args) {
```

Comment:
Prismic is a bit 🎺 in the sense that there's some repeat boilerplate (`Prismic.api(...)`, `.form('everything')`, '.ref(api.master()`, `.submit()`) that's required in order to make an API call.

You could abstract this boilerplate into its own function to make it a bit DRY'er. [I started on it](https://github.com/redbadger/badger-brain/blob/d1d631f538d945e111607edd5e5dab15ed4499d8/lib/fetch.js#L69) ages ago, but never got around to finishing it. This way, everything common is contained in a function that's used across every context where an API request is required.